Detectors/TPC/workflow/include/TPCWorkflow/TPCFourierTransformAggregatorSpec.h

```diff
@@ -179,19 +244,165 @@ class TPCFourierTransformAggregatorSpec : public o2::framework::Task
     mIntervalsBuffer.resize(expectedLanes);
     mIDCOneBuffer.resize(expectedLanes);
   }
+
+  void makeTPCScaler(DataAllocator& output, const bool eos)
+  {
+    LOGP(info, "Making TPC scalers");
+    // check if IDC scalers can be created - check length of continous received IDCs
+    std::vector<std::pair<long, long>> times;
+    times.reserve(mTPCScalerCont.idcs.size());
+    for (const auto& idc : mTPCScalerCont.idcs) {
+      times.emplace_back(idc.second.timestamp[0], idc.second.timestamp[1]);
```

Comment:
btw, your mTPCScalerCont is an unordered_map, in looping in this way, don't you risk to upload not in time ordered way? I think std::map would be safer.

Reply:
thats why I fill the time information to the vector and sort this vector afterward and then loop over the sorted vector.